contrib/runit/run

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+exec chpst -u loki:loki /usr/local/bin/lokinet ~loki/.lokinet/lokinet.ini 2>&1
```

Comment:
shouldn't it be `~/.lokinet/lokinet.ini` instead of `~loki/.lokinet/lokinet.ini` or is this some runit-ism?

Reply:
It's a chpst-ism because home dir is still /root I think.

Reply:
i see